k8s/dns-ssl-kube-lego/README.md

```diff
@@ -0,0 +1,215 @@
+# Endpoints DNS with SSL certificate issued by Let's Encrypt through Kube-Lego
+
+This example demonstrates how to deploy an Endpoints service on GKE with automatic
+provisioning of an SSL certificate for the .cloud.goog service domain.
+
+## Requirements
+
+For this example, you need:
+
+ -  A proper installation of the Cloud SDK, configured with an account and a project:
+    ```bash
+    gcloud config list
+    # [core]
+    # account = ...
+    # project = ...
+    ```
+    Make sure your gcloud installation is up-to-date:
+    ```bash
+    gcloud components update
+    ```
+    Make sure your credentials are up-to-date:
+    ```bash
+    gcloud auth login
+    ```
+- A proper GKE cluster created in your project:
+    ```bash
+    CLUSTER_NAME=endpoints-dns-sample
+    CLUSTER_ZONE="us-west1-a"
+    gcloud container clusters create ${CLUSTER_NAME} --zone=${CLUSTER_ZONE} --num-nodes=3
```

Comment:
Need to enable the Container Engine API before this can be run.

Reply:
Added `gcloud services enable container.googleapis.com`